torch/testing/_internal/common_utils.py

```diff
@@ -2375,3 +2376,22 @@ def disable_gc():
             gc.enable()
     else:
         yield
+
+def has_breakpad():
+    # In CI we expect these builds to have breakpad included, if they don't
+    # then something is probably wrong with the build and we need the test
+    # to fail in order to be alerted
+    job_name = os.getenv("CIRCLE_JOB", os.getenv("BUILD_ENVIRONMENT", ""))
+    prefixes = [
+        "pytorch_linux_bionic_py3_8_gcc9_coverage",
+        "binary_linux_manywheel",
+    ]
+    if any([prefix in job_name for prefix in prefixes]):
+        return True
```

Comment:
Can you please delete those (we should not hardcode any assumptions about CI in torch/testing folder